.rubocop.yml

```diff
@@ -31,3 +31,9 @@ Lint/BlockAlignment:
 
 Lint/InheritException:
   Enabled: false
+
+Style/GuardClause:
+  Enabled: false
```

Comment:
I like it in some cases, but not in others. And Rubocop was forcing me to use it in places where I definitely didn't want to use them.